DDCore/include/XML/UnicodeValues.h

```diff
@@ -192,9 +192,11 @@ UNICODE (global);
 UNICODE (global_grid_xy);
 UNICODE (grid_size_x);
 UNICODE (grid_size_y);
-UNICODE (grid_size_z);
+UNICODE (grid_size_u);
+UNICODE (grid_size_v);
 UNICODE (grid_size_phi);
 UNICODE (grid_xyz);
+UNICODE (grid_angle);
```

Comment:
Maybe this should be placed more alphabetically sorted?

Reply:
I rewrote the patch so that it's alphabetically sorted.

Reply:
`grid_angle` should still come before `grid_size...`